README.md

```diff
@@ -21,6 +21,11 @@ are either old, abandoned, hosted on CVS, written in platform-specific C, etc.
 Rust provides a good, platform-agnostic way of writing systems utils that are easy
 to compile anywhere, and this is as good a way as any to try and learn it.
 
+Rust Version
+------------
+
+coreutils supports and is tested on version `1.20.0` and above (including beta and nightly builds) of `rustc`.
```

Comment:
`coreutils` should probably be changed to `uutils` to keep consistent with the rest of the README.

Reply:
Changed.

Should the same be done here? https://github.com/uutils/coreutils#contribute

Reply:
Ah, I never noticed that.  Yes, please change that one as well.